packages/fxa-payments-server/src/components/NewUserEmailForm/index.tsx

```diff
@@ -16,14 +16,18 @@ import './index.scss';
 import { config } from '../../lib/config';
 
 export type NewUserEmailFormProps = {
-  getString: (id: string) => string;
+  getString?: (id: string) => string;
+  setValidEmailSet: Function;
```

Comment:
Why not set the actual email?  We do need to use it in a request.  Is there any way to get the email address instead?